org.speed_dreams.SpeedDreams.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Speed Dreams
+Comment=The Open Racing Car Simulator Fork
+Icon=org.speed_dreams.SpeedDreams
+Exec=speed-dreams-2
+Categories=Game;Application;
```

Comment:
Relevant subcategory should also be added: _Application;Game;**SportsGame;**_

Reply:
Done